code/game/objects/items/weapons/trophy.dm

```diff
@@ -0,0 +1,17 @@
+/obj/item/weapon/trophy
+  name = "hunting trophy"
+  desc = "You should never see this."
+  icon = 'icons/obj/trophies.dmi'
+  var/id = ""
```

Comment:
since the ID is always the same as the icon_state (IMO, a good thing), then this var is pointless as you can just use the trophy's icon state to tell the frame what it should look like.